Apps/WebClient/src/ClientApp/src/views/pcrTest.vue

```diff
@@ -396,6 +399,7 @@ export default class PcrTestView extends Vue {
                         this.displaySuccess();
                     })
                     .catch((err: ResultError) => {
+                        this.clearError();
```

Comment:
We also need to clear the errors when the submission is successful.  It might be easiest to do a single call at the start of the function.